modules/karma.py

```diff
@@ -8,7 +8,7 @@
 WORD_STOP = [",", ":"]
 KARMA_DELAY_SECONDS = 3
 
-REGEX_KARMA = re.compile(r"^(?:(\S+:) )?(.*)(\+{2}|\-{2})$")
+REGEX_KARMA = re.compile(r'(?:\((.+?)\)|(\S+))(\+\+|--)(\s+|$)')
```

Comment:
`r'(?:\(([^)]+)\)|(\S+))(\+\+|--)(\s+|$)'` so we don't need to break out the non-greedy

Reply:
Done!